log/log.go

```diff
@@ -45,7 +45,7 @@ func NewLogger(config config.Log) (*zap.Logger, error) {
 	return logger, nil
 }
 
-func HttpRequestToLogFields(r *http.Request) []zapcore.Field {
+func HTTPRequestToLogFields(r *http.Request) []zapcore.Field {
```

Comment:
__[golint]__
* [golint] exported function HTTPRequestToLogFields should have comment or be unexported

<!-- Comment by [SideCI](https://sideci.com) -->